### ISSUE TITLE
Error on startup: Unable to parse JSON payload from GitHub

### ISSUE THREAD

**Issue by KrisAphalon** [RPTools/maptool#2263]
**Describe the bug**
Opening MapTool creates an error:
```
ERROR net.rptools.maptool.client.AppUpdate - Unable to parse JSON payload from GitHub...
java.lang.IllegalStateException: Not a JSON Object: "{ [...]
```
Software runs fine afterwards.

**To Reproduce**
Make sure you have access to internet

1. Open the MapTool using console
2. See the error while opening
or
1. Open the MapTool without console
2. See the error in log file

**Expected behavior**
Opening MapTool should not create an error (possibly do what it is intended to, not just hide it)

**MapTool Info**
- Version: 1.7.0
- Install: Upgrade from 1.6.1, using .deb package

**Desktop (please complete the following information):**
 - OS: Linux - Sparky Linux (debian-based)
 - Version 5.9.0-sparky-amd64
 - `name -a`: `Linux Aphalon 5.9.0-sparky-amd64 #1 SMP Mon Oct 12 10:47:01 CEST 2020 x86_64 GNU/Linux`

**Additional context**
[maptool.log](https://github.com/RPTools/maptool/files/5395891/maptool.log)
[debug_information.log](https://github.com/RPTools/maptool/files/5395893/debug_information.log)

**Comment by Azhrei** [comment]
This is a known issue regarding the GitHub payload returned during a check to see if an upgrade is available, but I don't see in reported elsewhere, so thank you for filing the bug report.

(The name is "MapTool", without the "s" on the end.)

**Comment by Phergus** [comment]
Fixed under #2393.
